sfc/visitors.html

```diff
@@ -0,0 +1,61 @@
+---
+layout: default
+title: Welcome Ingress Visitors!
+header: Welcome Ingress Visitors!
+---
+  <p>
+    Welcome to the Visitor area for the San Francisco Resistance team.
+  </p>
+
```

Comment:
Might be nice to link back to the sfc and/or the main index page from this one.

We've traditionally not needed much nav since most of the links were just redirects, but we can start thinking about that if you want to add more content to this site.